libraries/lib-effects/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ set( LIBRARIES
    lib-command-parameters-interface
    lib-numeric-formats-interface
    lib-realtime-effects
+   lib-stretching-sequence-interface
```

Comment:
This line, and the one in src/CMakeLists.txt, are unnecessary for the build, and I would prefer you omit them so that the generated library graph does not have unnecessary edges.